### ISSUE TITLE
Segfault in static build 0.7.1-1

### ISSUE THREAD

**Issue by rekcuFniarB** [Alexey-Yakovenko/deadbeef#1555]
Segmentation fault when trying to play anything. Version 0.7.1-1_x86-64 static (from sourceforge.net) on Ubuntu 15.10.

Output:
```
Segmentation Fault
backtrace() returned 16 addresses
/opt/deadbeef/deadbeef-0.7.1/deadbeef() [0x403ab3]
/lib/x86_64-linux-gnu/libc.so.6(+0x352f0) [0x7f1d96d252f0]
/lib/x86_64-linux-gnu/libc.so.6(strrchr+0x25) [0x7f1d96d7d395]
/opt/deadbeef/deadbeef-0.7.1/deadbeef() [0x4301ce]
/opt/deadbeef/deadbeef-0.7.1/deadbeef() [0x430460]
/opt/deadbeef/deadbeef-0.7.1/plugins/ddb_gui_GTK2.so(gtkui_set_titlebar+0x8a) [0x7f1d966718fa]
/opt/deadbeef/deadbeef-0.7.1/plugins/ddb_gui_GTK2.so(+0x32a17) [0x7f1d96671a17]
/lib/x86_64-linux-gnu/libglib-2.0.so.0(g_main_context_dispatch+0x15a) [0x7f1d94ea0eaa]
/lib/x86_64-linux-gnu/libglib-2.0.so.0(+0x4a250) [0x7f1d94ea1250]
/lib/x86_64-linux-gnu/libglib-2.0.so.0(g_main_loop_run+0xc2) [0x7f1d94ea1572]
/usr/lib/x86_64-linux-gnu/libgtk-x11-2.0.so.0(gtk_main+0xb7) [0x7f1d96123a07]
/opt/deadbeef/deadbeef-0.7.1/plugins/ddb_gui_GTK2.so(gtkui_thread+0x71f) [0x7f1d96672bcf]
/opt/deadbeef/deadbeef-0.7.1/plugins/ddb_gui_GTK2.so(+0x33e39) [0x7f1d96672e39]
/opt/deadbeef/deadbeef-0.7.1/deadbeef() [0x403776]
/lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf0) [0x7f1d96d10a40]
/opt/deadbeef/deadbeef-0.7.1/deadbeef() [0x4039b8]
```
GDB:
```
Program received signal SIGSEGV, Segmentation fault.
strrchr () at ../sysdeps/x86_64/strrchr.S:32
32      ../sysdeps/x86_64/strrchr.S: Нет такого файла или каталога.
(gdb) bt
#0  strrchr () at ../sysdeps/x86_64/strrchr.S:32
#1  0x00000000004301ce in ?? ()
#2  0x0000000000430460 in ?? ()
#3  0x00007ffff6c668fa in gtkui_set_titlebar () from /opt/deadbeef/deadbeef-0.7.1/plugins/ddb_gui_GTK2.so
#4  0x00007ffff6c66a17 in ?? () from /opt/deadbeef/deadbeef-0.7.1/plugins/ddb_gui_GTK2.so
#5  0x00007ffff5495eaa in g_main_context_dispatch () from /lib/x86_64-linux-gnu/libglib-2.0.so.0
#6  0x00007ffff5496250 in ?? () from /lib/x86_64-linux-gnu/libglib-2.0.so.0
#7  0x00007ffff5496572 in g_main_loop_run () from /lib/x86_64-linux-gnu/libglib-2.0.so.0
#8  0x00007ffff6718a07 in gtk_main () from /usr/lib/x86_64-linux-gnu/libgtk-x11-2.0.so.0
#9  0x00007ffff6c67bcf in gtkui_thread () from /opt/deadbeef/deadbeef-0.7.1/plugins/ddb_gui_GTK2.so
#10 0x00007ffff6c67e39 in ?? () from /opt/deadbeef/deadbeef-0.7.1/plugins/ddb_gui_GTK2.so
#11 0x0000000000403776 in ?? ()
#12 0x00007ffff7305a40 in __libc_start_main (main=0x402c80, argc=1, argv=0x7fffffffdd38, init=<optimized out>, 
    fini=<optimized out>, rtld_fini=<optimized out>, stack_end=0x7fffffffdd28) at libc-start.c:289
#13 0x00000000004039b8 in ?? ()
```

**Comment by Alexey-Yakovenko** [comment]
Please attach your deadbeef config file.

**Comment by Alexey-Yakovenko** [comment]
ALso, please check if 0.7.2-rc6 works: https://sourceforge.net/projects/deadbeef/files/travis/linux/0.7/deadbeef-static_0.7.2-rc6-1_x86_64.tar.bz2/download

**Comment by rekcuFniarB** [comment]
[deadbeef config file](https://github.com/Alexey-Yakovenko/deadbeef/files/233506/deadbeef.conf.txt)

0.7.2-rc6 works.

**Comment by Alexey-Yakovenko** [comment]
The issue is caused by a bug in %filename% code, which is already fixed in 0.7.2-rc*.
